src/sdk/object_sdk.js

```diff
@@ -207,7 +207,16 @@ class ObjectSDK {
     async load_requesting_account(req) {
         try {
             const token = this.get_auth_token();
-            if (!token) return;
+            const is_anon = !(token && token.access_key);
+            if (is_anon) {
+                this.requesting_account = {
+                    nsfs_account_config: {
+                        uid: process.getuid(),
+                        gid: process.getgid(),
+                    }
+                };
```

Comment:
I think this is unsafe - we are creating a "partial" account for this request, and also letting it use the process elevated FS permissions.

I suggest a different approach - how about that all anonymous access will look up a "fixed" account that would have to be created in the system to allow annonymous access. That account will not have access keys off course, but will have its own nsfs_account_config provided so that it can be configured to run as a regular user on the FS. We need to hash out the details of that, but I think this approach will provide much more flexibility and security. 

WDYT @naveenpaul1 @romayalon @madhuthorat ?